libs/payments/cart/src/lib/cart.manager.ts

```diff
@@ -33,11 +33,12 @@ import type { AccountDatabase } from '@fxa/shared/db/mysql/account';
 // For an action to be executed, the cart state needs to be in one of
 // valid states listed in the array of CartStates below
 const ACTIONS_VALID_STATE = {
-  updateFreshCart: [CartState.START],
+  updateFreshCart: [CartState.START, CartState.PROCESSING],
```

Comment:
Should I use `updateCart` from processing? what's the right method to use here?

Reply:
Because we set the cart state to processing first, and then second call the payWithStripe action, `updateFreshCart` gets called when the cart is in the processing state

Reply:
Ah I see. I believe that's an oversight and `updateFreshCart` should not be used here.

No action required in this PR. This can be fixed in a polish PR/fast follows.

Just to explain the thinking, the intention with Cart Manager is to make available specific actions allowed to the Cart table, under specific circumstances. The idea being that this will allow for easier maintenance and general understanding when working with the Cart.